pkg/operator/configobservation/etcd/observe_etcd.go

```diff
@@ -446,8 +439,17 @@ func (e *etcdObserver) setObservedEtcdFromEndpoint(bucket string) error {
 			status := ceoapi.MemberUnknown
 			switch {
 			case isPodCrashLoop(bucket, pod):
```

Comment:
lets just make remove this case for now IMO.

We check if we have a member pending here. I believe that would be true if the readiness probe was failing on the crashloop pod.

func (c *ClusterMemberController) isClusterEtcdOperatorReady() bool {